lib/cli/commands/validate.js

```diff
@@ -30,8 +30,19 @@ module.exports = function (opts) {
         nullable: true
       })
 
+      let { support } = userPackageJson
+
+      if (support === true) {
+        support = await fs.readJSON(path.join(process.cwd(), 'package-support.json'))
+      } else if (typeof support === 'string') {
+        support = await fs.readJSON(path.join(process.cwd(), support))
```

Comment:
this doesn’t validate that it’s a path within the package - ie, it doesn’t prevent ..-ing out

Reply:
Should be fixed.

Reply:
I had more completely addressed that in my PR (And now the code have been moved as well so there will likely be conflict)